.env.template

```diff
@@ -1,8 +1,12 @@
 # URL to use for fetcing IPFS content. Set to https://drips.mypinata.cloud to use public Drips gateway.
 PUBLIC_PINATA_GATEWAY_URL=string
 
-# Infura key for connecting to Ethereum networks. You can create a free Infura account for development.
+# Optional Infura key.
 INFURA_KEY=string
+# Optional Alchemy key.
+ALCHEMY_KEY=string
+# Optional Filecoin key.
+FILECOIN_KEY=string
```

Comment:
could you just add a sentence explaining that values for at least one of the providers are required?